### PR TITLE
ZenDesk widget

### PR DESCRIPTION
**Pull request by Howesy77**

### CODE REVIEW

File: src/SFA.DAS.ProviderApprenticeshipsService.Web/Views/Shared/_ZenDeskWidget.cshtml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+                filter: {
+                    section: @Html.GetZenDeskSnippetSectionId()
+                }
+            }
+        }
+    };
+</script>
+<script id="ze-snippet" src="https://static.zdassets.com/ekr/snippet.js?key=@Html.GetZenDeskSnippetKey()"></script>
+<script id="co-snippet" src="https://embed-euw1.rcrsv.io/@Html.GetZenDeskCobrowsingSnippetKey()?zwwi=1"></script>
+@Html.SetZendeskLabels($"matp-{label}")
```
Comment (cofaulco) [inline src/SFA.DAS.ProviderApprenticeshipsService.Web/Views/Shared/_ZenDeskWidget.cshtml:28]: not sure if the prefix can be applied globally here? Will that be the prefix for the "Organisations" pages and other Account related pages in PAS?

File: src/SFA.DAS.ProviderApprenticeshipsService.Web/Extensions/HtmlHelperExtensions.cs [hunk 2]
```diff
@@ -83,5 +84,48 @@ public static MvcHtmlString GetClientContentByType(this HtmlHelper html, string
             var content = userResponse;
             return MvcHtmlString.Create(content.Content);
         }
+
+        public static MvcHtmlString SetZendeskLabels(this HtmlHelper html, params string[] labels)
+        {
+            var apiCallString = "<script type=\"text/javascript\">zE('webWidget', 'helpCenter:setSuggestions', { labels: [";
+
+            var first = true;
+            foreach (var label in labels)
```
Comment (cofaulco) [inline src/SFA.DAS.ProviderApprenticeshipsService.Web/Extensions/HtmlHelperExtensions.cs:93]: I think the code implemented in EAS is a bit cleaner, and might be good to keep it consistent:
https://github.com/SkillsFundingAgency/das-employerapprenticeshipsservice/blob/a59dd8ffbe9f630bd26aff947fdad96a9f5fd23a/src/SFA.DAS.EmployerAccounts.Web/Extensions/HtmlHelperExtensions.cs#L51-L60

Might be worth updating the confluence page with the alternate approach

File: src/SFA.DAS.ProviderApprenticeshipsService.Web/Views/Shared/_ZenDeskWidget.cshtml (hunk 1 above)
Comment (Howesy77) [reply src/SFA.DAS.ProviderApprenticeshipsService.Web/Views/Shared/_ZenDeskWidget.cshtml]: Removed prefix

File: src/SFA.DAS.ProviderApprenticeshipsService.Web/Extensions/HtmlHelperExtensions.cs (hunk 2 above)
Comment (Howesy77) [reply src/SFA.DAS.ProviderApprenticeshipsService.Web/Extensions/HtmlHelperExtensions.cs]: Changed to EAS version

**Review by cofaulco — APPROVED** [review]

**Pull request merged**